lib/screens/explore_screens/regions_screens/regions_main/components/region_event_card/interactor/usecases/get_n_profiles_use_case.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:seeds/datasource/remote/api/profile_repository.dart';
+import 'package:seeds/datasource/remote/model/profile_model.dart';
+import 'package:seeds/domain-shared/base_use_case.dart';
+
+class GetNProfilesUseCase extends InputListUseCase<ProfileModel, _Input> {
+  static _Input input(List<String> accounts, int requiredCalls) => _Input(accounts, requiredCalls);
+
+  @override
+  Future<List<Result<ProfileModel>>> run(_Input input) async {
```

Comment:
I think this should be a `Future<Result<List<ProfileModel>>>`

InputListUseCase is not really needed. 

You can map the list of results coming from the server into a Result<List< ProfileModel>>